### ISSUE TITLE
Font scaling in editor is off

### ISSUE THREAD

**Issue by anthonyajumelles** [kmwallio/ThiefMD#76]
Here's a good example. On the left is Gnome Terminal and both windows have Overpass Mono at size 14 selected.

![Screenshot from 2020-10-17 18-02-08](https://user-images.githubusercontent.com/62868183/96354315-624f2600-10a3-11eb-940e-e227f76ada39.png)

**Comment by kmwallio** [comment]
Thanks for reporting the issue.

Can you let me know your monitor resolution? I tried to convert pt to rem (not sure if this is needed), but I think the conversion might be different for HiDPI vs non-HiDPI monitors (and I guess GNOME/GTK offers it's own scaling, so maybe I don't need the conversion at all).

**Comment by kmwallio** [comment]
I've fixed this in the beta branch (f1d4e64fe52b473bc3021f713a1eeb0ba25ad7e3). There's still some scaling when switching to full screen. I might need to tweak how much.

Here's a comparison with ThiefMD and the Terminal set to 12pt.
![Fixed font scaling](https://user-images.githubusercontent.com/132455/96376846-2b762000-1136-11eb-902e-cd9392647a1a.png)

I changed the default pt size to 14 which seems close to what the default scaled 12pt was.
